src/search.cpp

```diff
@@ -100,10 +100,7 @@ namespace {
     return VALUE_DRAW - 1 + Value(thisThread->nodes & 0x2);
   }
 
-  // Skill structure is used to implement strength limit. If we have an uci_elo then
-  // we convert it to a suitable fractional skill level using anchoring to CCRL Elo
-  // (goldfish 1.13 = 2000) and a fit through Ordo derived Elo for a match (TC 60+0.6)
-  // results spanning a wide range of k values.
+  // Skill structure is used to implement strength limit
```

Comment:
This is not what @dav1312 wanted. The skill is no longer anchored to goldfish but instead stash was used. See https://github.com/vondele/Stockfish/commit/a08b8d4e9711c20acedbfe17d618c3c384b339ec

Reply:
@Disservin I see, what do you think about the following update?

// Skill structure is used to implement strength limit. If we have a UCI_Elo, we now convert it to a suitable fractional skill level using anchoring to the Stash engine.
// This update is based on a fit of the Elo results for games played between the master at various skill levels and various versions of the Stash engine, all ranked at CCRL.
// Skill 0 .. 19 now covers CCRL Blitz Elo from 1320 to 3190, approximately.

Reply:
>  suitable fractional skill level using anchoring to the Stash engine.

this sounds a bit weird?

Reply:
// Skill structure is used to implement strength limit. If we have a UCI_Elo, we now convert it to an appropriate skill level using anchoring to the Stash engine.
// This method is based on a fit of the Elo results for games played between the master at various skill levels and various versions of the Stash engine, all ranked at CCRL.
// Skill 0 .. 19 now covers CCRL Blitz Elo from 1320 to 3190, approximately.